### ISSUE TITLE
Dumb question: how to install

### ISSUE THREAD

**Issue by siyu6974** [wookayin/alfred-latex-symbols-workflow#5]
Hi, 

Please forgive me for this dumb question, but I did `npm install --global alfred-latex-symbols` and nothing shown up in Alfred Workflow. Doing `tex` in Alfred also does not work. Am I missing something here?

Thank you

**Comment by wookayin** [comment]
@siyu6974 Hi! After the installation, you should be able to see a symbolic link generated in the workflow folder:

```
ls -alh /path/to/Alfred/Alfred.alfredpreferences/workflows 
```

Do you see `alfred-latex-symbols` there? It should be linked to a node modules directory, e.g., `/opt/homebrew/lib/node_modules/alfred-latex-symbols`.

Also, what does `npm list --global` print for you?

**Comment by siyu6974** [comment]
Hi wookayin,

I don't see `alfred-latex-symbols` in `Alfred.alfredpreferences/workflows` after `npm install --global`. And for some reason I can't reproduce the case again. Some weird permission problem appear and I got the following error.

```
❯ sudo npm install --global alfred-latex-symbols

npm WARN deprecated xmldom@0.1.31: Deprecated due to CVE-2021-21366 resolved in 0.5.0
npm ERR! code 1
npm ERR! path /usr/local/lib/node_modules/alfred-latex-symbols
npm ERR! command failed
npm ERR! command sh -c alfy-init
npm ERR! Error: Command failed with exit code 77: alfred-link
npm ERR! You are not allowed to run this app with root permissions.
npm ERR! If running without sudo doesn't work, you can either fix your permission problems or change where npm stores global packages by putting ~/npm/bin in your PATH and running:
npm ERR! npm config set prefix ~/npm
npm ERR!
npm ERR! See: https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md
npm ERR!     at makeError (/usr/local/lib/node_modules/alfred-latex-symbols/node_modules/execa/lib/error.js:60:11)
npm ERR!     at handlePromise (/usr/local/lib/node_modules/alfred-latex-symbols/node_modules/execa/index.js:118:26)
npm ERR!     at processTicksAndRejections (internal/process/task_queues.js:93:5)
npm ERR!     at async file:///usr/local/lib/node_modules/alfred-latex-symbols/node_modules/alfy/init.js:11:3 {
npm ERR!   shortMessage: 'Command failed with exit code 77: alfred-link',
npm ERR!   command: 'alfred-link',
npm ERR!   escapedCommand: 'alfred-link',
npm ERR!   exitCode: 77,
npm ERR!   signal: undefined,
npm ERR!   signalDescription: undefined,
npm ERR!   stdout: '',
npm ERR!   stderr: 'You are not allowed to run this app with root permissions.\n' +
npm ERR!     "If running without sudo doesn't work, you can either fix your permission problems or change where npm stores global packages by putting ~/npm/bin in your PATH and running:\n" +
npm ERR!     'npm config set prefix ~/npm\n' +
npm ERR!     '\n' +
npm ERR!     'See: https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md',
npm ERR!   failed: true,
npm ERR!   timedOut: false,
npm ERR!   isCanceled: false,
npm ERR!   killed: false
npm ERR! }

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/siyu/.npm/_logs/2022-05-03T00_06_12_277Z-debug-0.log
```

Fortunately, base on your reply, I `cd` into `Alfred/Alfred.alfredpreferences/workflows ` and do the `npm install`. And everything starts working!

**Comment by wookayin** [comment]
I see, so `npm install --global` failed due to permission issue. Yes, you should have noticed the error message. Please read the error message carefully.

> npm ERR!   stderr: 'You are not allowed to run this app with root permissions.\n' +

Actually you should **not** run the command with **sudo**, it will cause other permission problems too. My recommmendation is to change the owner of the directory:

```
chown $USER /usr/local/lib/node_modules/
```
